front/app/containers/ProjectsShowPage/timeline/index.tsx

```diff
@@ -97,35 +101,47 @@ const ProjectTimelineContainer = memo<Props>(({ projectId, className }) => {
     setPhaseURL(phase.id, phases.data, project.data);
   };
 
-  if (project && selectedPhase) {
+  if (project && selectedPhase && !isNilOrError(currentLocale)) {
     const selectedPhaseId = selectedPhase.id;
     const participationMethod = selectedPhase.attributes.participation_method;
-    const isPastPhase =
-      pastPresentOrFuture(selectedPhase.attributes.end_at) === 'past';
-
+    const isPastPhase = selectedPhase.attributes.end_at
+      ? pastPresentOrFuture(selectedPhase.attributes.end_at) === 'past'
+      : false;
```

Comment:
Possible simplification?

```suggestion
        const isPastPhase = selectedPhase.attributes.end_at && pastPresentOrFuture(selectedPhase.attributes.end_at) === 'past'
```

Reply:
Good point, changing it to `!!selectedPhase.attributes.end_at && pastPresentOrFuture(selectedPhase.attributes.end_at) === 'past'` to keep it a boolean